spec/factories/document_factory.rb

```diff
@@ -4,7 +4,6 @@
   factory :document do
     content_id { SecureRandom.uuid }
     locale { I18n.available_locales.sample }
-    document_type_id { build(:document_type, path_prefix: "/prefix").id }
```

Comment:
Tbh I think this commit could be fixed up to the first one too. Each commit should be in a state where its tests pass; I suspect the first few commits don't and are only now beginning to pass with these test tweaks.

I don't think it would be too much code, if that's what you're trying to avoid here, and in a case where you're mostly moving things from one class to another I think it would be clearer to have it all in one place 🤔